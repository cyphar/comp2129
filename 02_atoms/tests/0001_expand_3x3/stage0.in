START 2 3 3
PLACE 1 0
DISPLAY
PLACE 2 2
DISPLAY
PLACE 2 0
DISPLAY
PLACE 2 1
DISPLAY
PLACE 0 1
DISPLAY
PLACE 2 1
DISPLAY
PLACE 0 2
DISPLAY
PLACE 1 2
DISPLAY
PLACE 1 1
DISPLAY
PLACE 1 2
DISPLAY
PLACE 1 1
DISPLAY
PLACE 2 2
