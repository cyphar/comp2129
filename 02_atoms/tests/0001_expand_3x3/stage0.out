Game Ready
Red's Turn

Green's Turn


+--------+
|  |R1|  |
|  |  |  |
|  |  |  |
+--------+

Red's Turn


+--------+
|  |R1|  |
|  |  |  |
|  |  |G1|
+--------+

Green's Turn


+--------+
|  |R1|R1|
|  |  |  |
|  |  |G1|
+--------+

Red's Turn


+--------+
|  |R1|R1|
|  |  |G1|
|  |  |G1|
+--------+

Green's Turn


+--------+
|  |R1|R1|
|R1|  |G1|
|  |  |G1|
+--------+

Red's Turn


+--------+
|  |R1|R1|
|R1|  |G2|
|  |  |G1|
+--------+

Green's Turn


+--------+
|  |R1|R1|
|R1|  |G2|
|R1|  |G1|
+--------+

Red's Turn


+--------+
|  |R1|R1|
|R1|  |G2|
|R1|G1|G1|
+--------+

Green's Turn


+--------+
|  |R1|R1|
|R1|R1|G2|
|R1|G1|G1|
+--------+

Red's Turn


+--------+
|  |R1|R1|
|R1|R1|G2|
|R1|G2|G1|
+--------+

Green's Turn


+--------+
|  |R1|R1|
|R1|R2|G2|
|R1|G2|G1|
+--------+

Green Wins!
